PureLISP.sh

```diff
@@ -22,34 +21,36 @@ cons () {
   eval CAR$CNUM=$1
   eval CDR$CNUM=$2
   CONSR=${CNUM}.conscell
-  CNUM=$((CNUM+1))
+  : $((CNUM++))
```

Comment:
It (and other same changes) does not work with "expecting primarry" message. `$((CNUM=+1)` has no one, but `CNUM`'s value is not being updated...

Reply:
I'm sorry for not haiving understood that increment and decriment are not POSIX-compliant; you are right. As you commented me, `: $((CNUM+=1))` or `CNUM=$((CNUM+1))` are what we were supposed to code. Also thank you for pointing out my mistake.